src/vm/CMakeLists.txt

```diff
@@ -27,6 +27,17 @@ if(CLR_CMAKE_PLATFORM_UNIX)
     add_compile_options(-fPIC)
 endif(CLR_CMAKE_PLATFORM_UNIX)
 
+if(FEATURE_GDBJIT)
+    set(VM_SOURCES_GDBJIT
+    ../gc/gccommon.cpp
```

Comment:
I also wonder why you need this change. The GC is now a separate static library.